File: src/fizzbuzz.c
#include <stdio.h>

int main() {
    int i = 1;
    for (i = 1; i <= 100; i++) {
        if (i % 3 == 0 && i % 5 == 0) {
            printf("fizzbuzz\n");
        } else if (i % 3 == 0) {
            printf("fizz\n");
        } else if (i % 5 == 0) {
            printf("buzz\n");
        }
    }
    return 0;
}

File: python/fizzbuzz.py
if __name__ == "__main__":
    for i in range(1,100):
        if i % 3 == 0 and i % 5 == 0:
            print str(i) + "fizzbuzz"
        elif i % 3 == 0:
            print str(i) + "fizz"
        elif i % 5 == 0:
            print str(i) + "buzz"


